lib/assembly-image/jp2_creator.rb

```diff
@@ -82,20 +82,15 @@ def layers
 
       KDU_COMPRESS_DEFAULT_OPTIONS = [
         '-num_threads 2', # forces Kakadu to only use 2 threads
-        '-precise', # forces the use of 32-bit representations
-        '-no_weights', # minimization of the MSE over all reconstructed colour components
         '-quiet', # suppress informative messages.
         'Creversible=no', # Disable reversible compression
-        'Cmodes=BYPASS', #
         'Corder=RPCL', # R=resolution P=position C=component L=layer
         'Cblk=\\{64,64\\}', # code-block dimensions; 64x64 happens to also be the default
         'Cprecincts=\\{256,256\\},\\{256,256\\},\\{128,128\\}', # Precinct dimensions; 256x256 for the 2 highest resolution levels, defaults to 128x128 for the rest
-        'ORGgen_plt=yes', # Insert packet length information
-        '-rate 1.5', # Ratio of compressed bits to the image size
+        '-rate -', # Ratio of compressed bits to the image size
         'Clevels=5' # Number of wavelet decomposition levels, or stages
```

Comment:
are these Kakadu options changes from @calavano and do they address issue #7 ?

Reply:
I'm not sure, this change came from Tony's previous PR that started this work

Reply:
I'm hoping Tony will comment when he gets a chance.